apps/condo/cypress/integration/pages/ticket.js

```diff
@@ -70,4 +70,87 @@ describe('Ticket',  function () {
+            cy.clearCookies()
+        })
+
+        describe('valid cases', () => {
+            const cases = [
+                './cypress/testFiles/ticket/import-ticket-success-1.xlsx',
+                './cypress/testFiles/ticket/import-ticket-success-2.xlsx',
+                './cypress/testFiles/ticket/import-ticket-success-3.xlsx',
+                './cypress/testFiles/ticket/import-ticket-success-4.xlsx',
+                './cypress/testFiles/ticket/import-ticket-success-5.xlsx',
+                './cypress/testFiles/ticket/import-ticket-success-6.xlsx',
+                './cypress/testFiles/ticket/import-ticket-success-7.xlsx',
+            ]
+
+            cases.forEach((filePath, index) => {
+                it(`can ticket import ${index + 1}`, () => {
+                    cy.task('keystone:createSupportWithProperty').then((response) => {
+                        authUserWithCookies(response)
+
+                        const ticketImport = new TicketImport()
+                        ticketImport
+                            .visit()
+                            .importTicketTable(filePath)
+                            .waitCreatingTicket()
+                            .closeSuccessModal()
+                    })
+                })
+            })
+        })
+
+        describe('invalid cases', () => {
+            const cases = [
+                './cypress/testFiles/ticket/import-ticket-error-1.xlsx',
+                './cypress/testFiles/ticket/import-ticket-error-3.xlsx',
+                './cypress/testFiles/ticket/import-ticket-error-5.xlsx',
+            ]
+
+            cases.forEach((filePath, index) => {
+                it(`can not ticket import ${index + 1}`, () => {
+                    cy.task('keystone:createSupportWithProperty').then((response) => {
+                        authUserWithCookies(response)
+
+                        const ticketImport = new TicketImport()
+                        ticketImport
+                            .visit()
+                            .importTicketTable(filePath)
+                            .waitFetchingContact()
+                            .closeErrorModal()
+                    })
+                })
+            })
+        })
+
+        describe('cases without required fields', () => {
+            const cases = [
+                './cypress/testFiles/ticket/import-ticket-error-2.xlsx',
+                './cypress/testFiles/ticket/import-ticket-error-4.xlsx',
+            ]
+
+            cases.forEach((filePath, index) => {
+                it(`can not ticket import ${index + 1}`, () => {
+                    cy.task('keystone:createSupportWithProperty').then((response) => {
+                        authUserWithCookies(response)
+
+                        const ticketImport = new TicketImport()
+                        ticketImport
+                            .visit()
+                            .importTicketTable(filePath)
+                            .closeErrorModal()
```

Comment:
is it possible to check the error details? or error rows or error message?

Reply:
maybe, i didn't tried